src/anyio/_backends/_asyncio.py

```diff
@@ -1865,6 +1866,9 @@ def __exit__(
     ) -> None:
         self._runner.__exit__(exc_type, exc_val, exc_tb)
 
+        if (send_stream := self._send_stream) is not None:
+            send_stream.close()
```

Comment:
It would be better to set up `.close()` as a "done callback" of the runner task, in `_call_in_runner_task()`.